### DISCUSSION TITLE
Set the default data-theme programmatically, not on <html> tag

### DISCUSSION THREAD

**Discussion by kobe-ra** [saadeghi/daisyui#2724] [Ideas / feature requests]
Hi, I'm trying to create a sveltekit capacitorjs native app.
I can't do `<html lang="sk" data-theme="business">` because apps on android and ios that use WebView don't have access to the html.
All my colors are now using `--fallback` colors, which is light by default and I'd need to overwrite them all. Is there some was to do this already? I haven't found it mentioned anywhere in the [docs](https://daisyui.com/docs/themes/).
Thank you.

**Comment by saadeghi** [comment]
You can use `data-theme` on any other element. If you don't have access to `<html>` tag, you can use it on the biggest container div you have access to.  

Also there's a [`themeRoot` config](https://daisyui.com/docs/config/) which you can choose a specific element that must receive the theme color variables.

Let me know if you have any questions

**Reply by kobe-ra** [reply]
This is how the apps look side by side with this code:

```
<div class="container" data-theme="business">
	...
	<button type="button" class="btn btn-primary">
		testb
	</button>
	<button type="button" class="btn btn-secondary" on:click={() => console.log('ay')}>
		testb
	</button>
</div>
```

I'm still trying to find more documentation about WebView, mainly if it is possible to use custom tag attributes, but if that's the case, that they only support the traditional attributes like `class, id, type, style` and so on, I'll need to find a different solution.

![image](https://github.com/saadeghi/daisyui/assets/32091524/06d5b583-ad04-4175-83c7-951b5b3d3180)

**Reply by saadeghi** [reply]
Looks like the browser on your simulator doesn't support `oklch` colors and it's using the fallback colors (which is the default light theme)

Can you see which version of chrome is that?

**Reply by kobe-ra** [reply]
I am very sorry for bothering you. The emulator had `webViewVersion: "103.0.5060.71"`. I just updated it to `webViewVersion: "120.0.6099.144"` and it's working properly. I'll buy you a coffee. Thank you for helping.
